sdk/communication/AzureCommunicationChat/Source/ChatThreadClient.swift

```diff
@@ -0,0 +1,348 @@
+    public func listMessages(
+        withOptions options: AzureCommunicationChatService.ListChatMessagesOptions? = nil,
+        completionHandler: @escaping HTTPResultHandler<PagedCollection<ChatMessage>>
+    ) {
+        service.listChatMessages(chatThreadId: threadId, withOptions: options) { result, httpResponse in
+            switch result {
+            case let .success(messages):
+                completionHandler(.success(messages), httpResponse)
+
+            case let .failure(error):
+                completionHandler(.failure(error), httpResponse)
+            }
+        }
+    }
+
+    /// Adds thread participants to a ChatThread. If the participants already exist, no change occurs.
+    /// - Parameters:
+    ///    - request : The request with thread participants to be added to the thread.
+    ///    - options: Add chat participants options.
+    ///    - completionHandler: A completion handler that receives a status code on success.
+    public func add(
+        chatParticipants: AddChatParticipantsRequest,
+        withOptions options: AzureCommunicationChatService.AddChatParticipantsOptions? = nil,
+        completionHandler: @escaping HTTPResultHandler<Void>
+    ) {
+        service
+            .add(
+                chatParticipants: chatParticipants,
+                chatThreadId: threadId,
+                withOptions: options
+            ) { result, httpResponse in
+                switch result {
+                case .success:
+                    completionHandler(.success(()), httpResponse)
+
+                case let .failure(error):
+                    completionHandler(.failure(error), httpResponse)
+                }
+            }
+    }
+
+    /// Removes a participant from the thread.
+    /// - Parameters:
+    ///    - chatMemberId : Id of the participant to remove.
+    ///    - options: Remove participant options
+    ///    - completionHandler: A completion handler that receives a status code on success.
+    public func remove(
+        chatParticipantId: String,
+        withOptions options: AzureCommunicationChatService.RemoveChatParticipantOptions? = nil,
+        completionHandler: @escaping HTTPResultHandler<Void>
+    ) {
+        service
+            .removeChatParticipant(
+                chatThreadId: threadId,
+                chatParticipantId: chatParticipantId,
+                withOptions: options
+            ) { result, httpResponse in
+                switch result {
+                case .success:
+                    completionHandler(.success(()), httpResponse)
+
+                case let .failure(error):
+                    completionHandler(.failure(error), httpResponse)
+                }
+            }
+    }
+
+    /// Gets the participants of the thread.
+    /// - Parameters:
+    ///    - options: List chat participants options.
+    ///    - completionHandler: A completion handler that receives the list of members on success.
+    public func listParticipants(
```

Comment:
in other places you refer to them as `chatParticipants`. It should stay consistent throughout--either `participants` or `chatParticipants`

Reply:
Updated, thanks!